src/digit/mod.rs

```diff
@@ -21,27 +26,55 @@ fn digits_convert<T: AsRef<str>>(src: T, from: Lang, to: Lang) -> String {
         .collect::<_>()
 }
 
+/// Takes a string which may contain English digits, and returns
+/// a string that represents the same digits but in Persian.
 pub fn digits_en_to_fa<T: AsRef<str>>(src: T) -> String {
     digits_convert(src, Lang::En, Lang::Fa)
 }
 
+/// Takes a string which may contain Persian digits, and returns
+/// a string that represents the same digits but in English.
 pub fn digits_fa_to_en<T: AsRef<str>>(src: T) -> String {
     digits_convert(src, Lang::Fa, Lang::En)
 }
 
+/// Takes a string which may contain Arabic digits, and returns
+/// a string that represents the same digits but in Persian.
+pub fn digits_ar_to_fa<T: AsRef<str>>(src: T) -> String {
+    digits_convert(src, Lang::Ar, Lang::Fa)
+}
+
+/// Takes a string which may contain Arabic digits, and returns
+/// a string that represents the same digits but in English.
+pub fn digits_ar_to_en<T: AsRef<str>>(src: T) -> String {
```

Comment:
I think it is a better idea to find the type of language in our function from the input content with regex

Reply:
What if it contains both Arabic and English?

Reply:
We give him an error

Reply:
But it's not an error :D